### ISSUE TITLE
Tutorial: Game is not paused when AI strategies screen is displayed

### ISSUE THREAD

**Issue by SimonThormeyer** [georgbuechner/fast_bsg#108]
On this screen the music (and hence the game) is not paused: 

<img width="687" alt="image" src="https://user-images.githubusercontent.com/49559340/215203356-e742389d-802d-4822-9c8f-766e502e4e76.png">

So if I take too much time on that screen, I lose immediately

**Comment by georgbuechner** [comment]
Yes, I will have to find a solution for this. There is currently no 

For now: either be quick or disable this feature in the settings :grimacing:  

Also, I changed the default settings so that ai-settings are not shown by default. 

[tbh this was originally just a debug print for me :D]
